amptorch/descriptor/Gaussian/calculate_sf.cpp

```diff
@@ -125,8 +127,10 @@ extern "C" int calculate_sf_cos(double** cell, double** cart, double** scale, in
     for (int i=0; i < 3; ++i) {
         // plane_d[i] / nbins[i] = height[i] / nbins[i] = size of bin in i dimension
         // cutoff * nbins[i] / plane_d[i] = cutoff / size of bin in i dimension = num bin needed in i dimension
+        // if cutoff < size: bin_range = 1, if cutoff > size: bin_range > 1
         bin_range[i] = ceil(cutoff * nbins[i] / plane_d[i]);
-        neigh_check_bins *= 2*bin_range[i];
+        // +1?
+        neigh_check_bins *= 2*bin_range[i]+1;
```

Comment:
This is just to make it bullet-proof. it's basically allocating space based on the radius. the original way should be fine, but I'm a little bit afraid about extreme cases, since it doesn't include local row/column. Thus +1 in each dimension to include local row/column should make it bullet-proof.